### PR TITLE
Addingpolishing admin controls

### LINKED ISSUE #59: Adding/Polishing Admin Controls

### PR DESCRIPTION
**Pull request by RobertDeWolf7**

### CODE REVIEW

File: src/main/resources/noUser/adminview.html [hunk 1]
```diff
@@ -6,6 +6,26 @@
...
+    border-radius: 0.3rem;
+    color: white;
+    background-color: #007bff;
+    border-color: #007bff;
+}
+.admin-table{
+    background-color: #d0deeb;
+}
+
+    </style>
```
Comment (pruthvichivukula) [inline src/main/resources/noUser/adminview.html]: should this be in a css file?

Comment (RobertDeWolf7) [reply src/main/resources/noUser/adminview.html]: We could make a change to the Spring Security Config files to allow css files to work, but as it stands right now, this is the simplest way to style

Comment (RobertDeWolf7) [reply src/main/resources/noUser/adminview.html]: Fixed issue, I was referencing wrong stylesheet

**Review by pruthvichivukula — APPROVED** [review]
my issue was addressed

**Pull request merged**